tests/login.test.js

```diff
@@ -70,3 +72,52 @@ describe("Initialize Bynder with permanent token", () => {
     expect(bynder.api.permanentToken).toEqual("test");
   });
 });
+
+
+describe("API call headers", () => {
+  describe("with permanent token", () => {
+    let mockedFunction = null;
+
+    beforeEach(() => {
+      bynder = new Bynder(configs);
+      bynder.api.permanentToken = "token";
+      mockedFunction = bynder.api._axios;
```

Comment:
I think another interesting strategy could be to mock the `axios` module, like described here: https://jestjs.io/docs/en/mock-functions#mocking-modules (coincidentally they have axios in their example as well :) )

That way, you don't have to alter the application code (with `this._axios`).

Reply:
Restored.